rules/isc-dhcp-server.mk

```diff
@@ -0,0 +1,15 @@
+# isc-dhcp-server
+
+ISC_DHCP_VERSION = 4.3.3
+ISC_DHCP_VERSION_FULL = ${ISC_DHCP_VERSION}-6
+
+export ISC_DHCP_VERSION ISC_DHCP_VERSION_FULL
+
+ISC_DHCP_SERVER = isc-dhcp-server_$(ISC_DHCP_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+$(ISC_DHCP_SERVER)_SRC_PATH = $(SRC_PATH)/isc-dhcp
```

Comment:
Since both `isc-dhcp-server` and `isc-dhcp-relay` have the same source package, I think it's best if they're both specified in the same rules file.

Reply:
Updated